funnel/views/event.py

```diff
@@ -108,14 +109,26 @@ def participant(profile, space):
         return jsonp(message="Unauthorized", code=401)
 
 
-@app.route('/<space>/event', subdomain='<profile>')
+def allowed_file(filename, allowed_exts=[]):
+    return '.' in filename and filename.rsplit('.', 1)[1] in allowed_exts
+
+
+@app.route('/<space>/event', methods=['GET', 'POST'], subdomain='<profile>')
 @lastuser.requires_login
 @load_models(
     (Profile, {'name': 'profile'}, 'g.profile'),
     ((ProposalSpace, ProposalSpaceRedirect), {'name': 'space', 'profile': 'profile'}, 'space'),
     permission='event-view')
 def events(profile, space):
-    return render_template('events.html', profile=profile, space=space, events=space.events.all())
+    attendee_sync_form = forms.Form()
+    if attendee_sync_form.validate_on_submit():
+        space_key = space.title.lower() + '-' + space.name
+        if space_key in app.config.get('EXPLARA_EVENT_IDS').keys():
+            ea = ExplaraAPI({'access_token': app.config.get('EXPLARA_ACCESS_TOKEN')})
+            tickets = ea.get_tickets(app.config.get('EXPLARA_EVENT_IDS').get(space_key))
```

Comment:
Move `EXPLARA_EVENT_IDS` and `EXPLARA_ACCESS_TOKEN` out of config and into the models.